bCNC.sh

```diff
@@ -2,6 +2,7 @@
 
 #Autodetect python version
 [ .$PYTHON = . ] && PYTHON=`which python2`
+[ .$PYTHON = . ] && PYTHON=`which python3`
```

Comment:
python2 is not really supported, so python3 should come first. can you please move it one line higher?